packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -1161,6 +1242,87 @@ class BreakpointsView implements CoreElementView {
   }
 }
 
+class PopupView extends CoreElement {
```

Comment:
can you add an integrate test for this functionality? There is already one for some debugger functionality.